### PR TITLE
new design for public notification

### PR DESCRIPTION
**Pull request by jmhauck**

### CODE REVIEW

File: src/components/public-notification/public-notification.tsx [hunk 1]
```diff
@@ -14,634 +14,661 @@
...
+     );
+   }
+ 
+   //--------------------------------------------------------------------------
+   //
+   //  Functions (private)
+   //
+   //--------------------------------------------------------------------------
+ 
+   _getActionGroup(
```
Comment (MikeTschudi) [inline src/components/public-notification/public-notification.tsx:184]: Would you please add function documentation to this file?

File: src/components/refine-selection/refine-selection.tsx [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+  }
+
+  //--------------------------------------------------------------------------
+  //
+  //  Functions (private)
+  //
+  //--------------------------------------------------------------------------
+
+
+  _modeChanged(evt: CustomEvent): void {
```
Comment (MikeTschudi) [inline src/components/refine-selection/refine-selection.tsx:153]: Similar request:  function doc, please

File: src/utils/publicNotificationUtils.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+ */
+
+/*
+ | Helper functions used in multiple components for public notification
+*/
+
+import { ISelectionSet } from "./interfaces";
+
+
+export function getSelectionIds(
```
Comment (MikeTschudi) [inline src/utils/publicNotificationUtils.ts:24]: Function doc, please

**Review by MikeTschudi — APPROVED** [review]
Nicely done--good looking UI, too!

One spelling change for UI and multiple undocumented functions.

**Comment by jmhauck** [comment]
spelling change and some doc updates...more doc work will need to be done but this takes care of a lot of the current ones I'm working on. Since this PR is so large want to go ahead and get it in so we can avoid conflict annoyance. Will do some more doc work once I'm back on components work

**Pull request merged**